recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "openmm-torch" %}
-{% set version = "0.2" %}
+{% set version = ".03" %} # Typo in tag name, oops!
```

I'd fix this in the `git_rev` field manually. Otherwise the package version metadata will not be sorted correctly!

```suggestion
{% set version = "0.3" %} # Typo in tag name, oops!
```

---

Oops, I didn't see this before merging.  Should I make another PR to fix it?

---

I am afraid that might be needed! Let's see how the current release shows up in Anaconda first.

---

It looks like it worked ok.  The correct version shows up at https://anaconda.org/conda-forge/openmm-torch.  I tried installing just to be sure, and it selected this version.